### PR TITLE
Feature 4

### PR DESCRIPTION
**Pull request by Namrata-J**
#4 here in this PR, i've added following features:
- header and sidebar responsiveness
- create post functionality
- added post card UI with all its related functionality i.e.
       1. like/dislike
       2. comment
       3. bookmark
       4. edit post
       5. follow/unfollow user
       6. delete post
- added comment related functionality
       1. upvote/downvote comment
       2. edit loggedIn user comment
       3. delete loggedIn user comment
- unfollowed users recommendation

### CODE REVIEW

**Review by simran61 — APPROVED** [review]
Looks Good

File: src/contexts/createPost-context.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    const [chosenEmojiForCommentBox, setChosenEmojiForCommentBox] = useState('');
+
+    const [showEmojiPicker, setShowEmojiPicker] = useState(false);
+
+    const onEmojiClick = (event, emojiObject) => {
+        setChosenEmoji(emojiObject.emoji);
+    };
+
+    const onCommentBoxEmojiClick = (event, emojiObject) => {
+        console.log(emojiObject)
```
Comment (dev-enforced) [inline src/contexts/createPost-context.js]: Can remove this debugger

**Review by dev-enforced — APPROVED** [review]
Live preview is working really welll. Good work on the styles Namrata

File: src/contexts/createPost-context.js (hunk 1 above)
Comment (Namrata-J) [reply src/contexts/createPost-context.js]: yes, thanks for pointing it out, will remove it!!

**Pull request merged**